src/pages/PhonesPage/PhonesPage.tsx

```diff
@@ -15,12 +15,16 @@ import bread__img from '../../assets/breadcrumbs-img/Breadcrumbs.png';
 import { SortType } from '@/enums/SortType';
 
 import styles from './styles.module.scss';
+import { Product } from '@/types/Product';
 
 export const PhonesPage = () => {
-  const [searchParams] = useSearchParams();
-  const { phones, loading, error } = usePhones();
+  const [searchParams, setSearchParams] = useSearchParams();
+  const { loading, error } = usePhones();
```

Comment:
Why do we need this?

Reply:
for pagination & dropdowns (it was Yevhenii's code)